tests/integration/fixtures/conference_article_expected.json

```diff
@@ -17,6 +17,7 @@
       "source":"submitter",
       "method":"submitter",
       "submission_number":"1",
+      "internal_uid":3,
```

Comment:
This test is brittle: if the ordering of the test changes, or some other users are created during a test run, the assertion that uses this static fixture is going to fail.

But I'm taking the decision to not compromise the code quality in favor of compromising the test quality until these tests are properly refactored.